### PR TITLE
[php-symfony5] New PHP Symfony 5 server generator

### PR DESCRIPTION
**Pull request by ybelenko**
I'm working on new PHP Symfony 5 server generator.

What I want to achieve with new generator:
1. Remove all source Symfony files from templates because Composer installation writes them automatically for us.
2. Change source files location, it slightly changed in Symfony 5.
3. Remove custom serializers and other helper classes and replace them with 3rd party packages.

Unfortunately, I have no time to work on it until September.

**Important! I gonna rebase and clean this PR few times. Changes are not final, don't review it unless you could do it again after cleanup. Just want to show current stage of development.**

Related issue #6522 

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
- [x] Run the shell script `./bin/generate-samples.sh`to update all Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. These must match the expectations made by your contribution. You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`
- [x] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

cc @jebentier @dkarlovi @mandrean @jfastnacht @ackintosh @renepardon
cc @reznikartem @FlaverSaver @xvilo

### CODE REVIEW

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfony5ServerCodegen.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```
Comment (wing328) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfony5ServerCodegen.java:3]: If it's written from scratch, you can remove line 3: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license

**Comment by coudenysj** [comment]
@ybelenko Is this still something you want to move forward with?

If you need any help, please let me know!

**Comment by ybelenko** [comment]
> @ybelenko Is this still something you want to move forward with?
> 
> 
> 
> If you need any help, please let me know!

As it turned out more difficult to get PR approved in this repo than submit PR itself. I have some free time right now, but PHP Slim4 server generator got higher priority for me personally. I would love to take care of other PHP based generators, but don't know when I can finally reach them(there is also my PHP data mocker repo and I want to develop it into useful package ASAP).

**Comment by ybelenko** [comment]
Closed in favour #11810

**Pull request closed without merging**